target/linux/mediatek/image/filogic.mk

```diff
@@ -769,6 +769,51 @@ endif
 endef
 TARGET_DEVICES += xiaomi_mi-router-wr30u-ubootmod
 
+define Device/xiaomi_mi-router-ax3000t
```

Comment:
Move those higher to keep alphabetical order, please

Reply:
Done